publications-2022.bib

```diff
@@ -29,4 +29,17 @@ @article{RoSchrWi2022
   url={https://doi.org/10.1007/s42452-022-04938-9}
 }
 
+@article{munch2022gc,
+  title={Efficient distributed matrix-free multigrid methods on locally refined meshes for FEM computations},
+  author= {Munch, Peter and Heister, Timo and Prieto Saavedra, Laura and Kronbichler, Martin},
+  year= {2022},
+  archivePrefix = {arXiv:2203.12292}
```

Comment:
Do you also have to add the doi for a link to appear?

Reply:
I think you want to write it like this entry:
```
  Url                      = {https://arxiv.org/abs/1907.06696}
```
The other arXiv preprints are all marked up as `@misc` and then run through this script:
```
   \begin{howpublished}  \howpublished\end{howpublished}<!--
    -->\begin{publisher}, \format[HTMLChars]{\publisher}\end{publisher}<!--
      -->\begin{address}, \format[HTMLChars]{\address}\end{address}<!--
        -->\begin{pages}, pp. \format[FormatPagesForHTML]{\pages}\end{pages}<!--
         -->\begin{note}  (\format[HTMLChars]{\note})\end{note}<!--
-->\begin{archivePrefix}  \format[HTMLChars]{\archivePrefix}\end{archivePrefix}<!--
 -->\begin{primaryClass}  \format[HTMLChars]{\primaryClass}:\end{primaryClass}<!--
       -->\begin{eprint}\format[HTMLChars]{\eprint}\end{eprint}<!--
         -->\begin{year}, \format[HTMLChars]{\year}\end{year}<!--
                     -->.
```
So that's the alternative. The arXiv website allows you to download a bibtex reference that matches this format.